### PR TITLE
Publish Puppet Platform 5.2 (HOLD FOR RELEASE)

### PR DESCRIPTION
**Pull request by gguillotte**
-   Publish Puppet 5.2.0 docs.
-   Publish Facter 3.9.0 docs.

## What to do:

**tl;dr:**

-   If changes are necessary to the Puppet 5.2 or Facter 3.9 docs PRs, the easiest option would be to merge my PRs anyway, then edit the content after merging.
-   If changes are necessary to the Puppet Server docs PR, the easiest option might be to have the server team merge the docs PR, make and commit the changes themselves if they're minor, and provide you with the new commit ID.
-   The docs won't build until Server/Platform Core provides a commit ID or tag that contains the merged docs PR and someone on docs adds that commit ID or tag to the `commit:` line for `puppetserver` in `_config.yml` (line 600).

----

**Long version:**

-   Confirm that the [Puppet 5.2 release notes and docs](https://github.com/puppetlabs/puppet-docs/pull/785) PR is accepted by someone from Platform Core or OS.
    -   (If any changes are necessary, the easiest route is to merge the PR anyway, then follow up with changes.)
-   Merge the Puppet 5.2 release notes PR.
-   Confirm that the Facter 3.9 release notes and core facts doc are accepted by someone from Platform OS.
    -   (If any changes are necessary, the easiest route is to merge the PR anyway, then follow up with changes.)
-   Merge the Facter 3.9 release notes PR.
-   Confirm that the Puppet Server 5.1 release notes and docs PR has been merged by the Server team.
-   After the Puppet, Facter, and Server PRs are merged, merge this PR.

The first docs build after merging this will probably fail because there is no commit ID or tag provided for puppetserver. I'd normally add this before publishing the docs, but I can't, and it's easier to follow up with another commit than it is to edit this PR or make a new one.

-   Get a commit ID from the puppetlabs/puppetserver repo from after the docs PR is merged. If you're not sure, ask the Puppet Server or Platform Core team to help you find an appropriate commit.
-   Edit `source/_config.yml` and place that commit ID as the value for the `commit` line of the `/puppetserver/5.1:` section in `_config.yml`.
-   Commit and push that change to master.

### CODE REVIEW

File: source/_config.yml [hunk 1]
```diff
@@ -579,6 +591,18 @@ documents:
     version: "1"
     nav: /_includes/hiera1.html
 
+  /puppetserver/5.1:
+    doc: puppetserver
+    version: "5.1"
+    nav: /puppet/5.2/_puppet_toc.html
+    external_source:
+      repo: git://github.com/puppetlabs/puppetserver.git
+      commit:
```
Comment (gguillotte) [inline source/_config.yml]: For the `commit`, get a commit ID from the puppet-server repo after the docs are merged ([PR #1506](https://github.com/puppetlabs/puppetserver/pull/1506)).
